pkg/config/common/common.go

```diff
@@ -146,8 +146,10 @@ type Configuration struct {
 	MaxOpenColumns              uint64         `yaml:"maxOpenColumns"`
 	UseNewPipelineConverted     bool
 	UseNewQueryPipeline         string `yaml:"isNewQueryPipelineEnabled"`
+	QueryTimeoutSecs            int    `yaml:"queryTimeoutSecs"` // query timeout in seconds
```

Comment:
nit: the name is already clear, so you don't really need this comment. You can keep it if you want though